### PR TITLE
Rename `Csa` to `Csb`

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr renames `Csa` to `Csb` for consistency with `Csw`, i.e. `Csb` (byte) -- `Csw` (word / wide), and to better reflect what it's actually doing.

**Test plan**

All builds are green.

**Closing issues**

...

### CODE REVIEW

File: librz/include/rz_bin.h [hunk 1]
```diff
@@ -162,13 +162,13 @@ typedef enum {
 
 enum {
 	RZ_BIN_STRING_ENC_DETECT = 'g',
-	RZ_BIN_STRING_ENC_ASCII = 'a',
+	RZ_BIN_STRING_ENC_ASCII = 'b',
```
Comment (XVilka) [inline librz/include/rz_bin.h]: What about renaming it as well?

File: librz/include/rz_util/rz_str.h [hunk 2]
```diff
@@ -16,12 +16,12 @@ typedef enum {
 } RzStrType;
 
 typedef enum {
-	RZ_STRING_ENC_LATIN1 = 'a',
+	RZ_STRING_ENC_LATIN1 = 'b',
```
Comment (XVilka) [inline librz/include/rz_util/rz_str.h]: And this one

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/include/rz_bin.h (hunk 1 above)
Comment (kazarmy) [reply librz/include/rz_bin.h]: Ok 436fbe4cb037eec35b75fc43c1e8924dc4d67d95

File: librz/include/rz_util/rz_str.h (hunk 2 above)
Comment (kazarmy) [reply librz/include/rz_util/rz_str.h]: Ok c7954897cb07803bcb941ee66e253cd98f62252a

**Review by XVilka — APPROVED** [review]
@wargio if you used `Csa` in jsdec, note this change.

**Pull request merged**